crates/figma_import/src/bin/dcf_info.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use bincode::Options;
+/// Utility program to fetch a doc and serialize it to file
+use clap::Parser;
+use figma_import::SerializedDesignDoc;
+
+#[derive(Debug)]
+struct ConvertError(String);
+impl From<bincode::Error> for ConvertError {
+    fn from(e: bincode::Error) -> Self {
+        eprintln!("Error during deserialization: {:?}", e);
+        ConvertError(format!("Error during deserialization: {:?}", e))
+    }
+}
+impl From<std::io::Error> for ConvertError {
+    fn from(e: std::io::Error) -> Self {
+        eprintln!("Error opening file: {:?}", e);
+        ConvertError(format!("Error opening file: {:?}", e))
+    }
+}
+
+#[derive(Parser, Debug)]
+struct Args {
+    #[arg(short, long)]
+    dcf_file: std::path::PathBuf,
```

Comment:
Can you set the argument to be positional so we don't need to specify --dcf-file? Just remove the short and long traits. It's the only argument so I don't think it makes sense to require the flag

Reply:
Done